src/prototype_creep_startup_tasks.js

```diff
@@ -117,6 +117,7 @@ Creep.recycleCreep = function(creep) {
 };
 
 Creep.getEnergy = function(creep) {
+  creep.pickupEnergy();
```

Comment:
If we want to add that, it should be in `Creep.prototype.getEnergy` and not hidden in the abstraction.

Reply:
it will be removed in next PR. `the hidden abstraction` we dont need. 
i dont like `Creep.getEnergy` and that `method.push` stuff.

Reply:
hm, if you touch the PR anyway, I would suggest to remove it, if it get's anyway removed.